src/semver/__about__.py

```diff
@@ -0,0 +1,8 @@
+__version__ = "3.0.0-dev.1"
```

Comment:
I think we should raise the development version:

```suggestion
__version__ = "3.0.0-dev.2"
```